src/gsi/gsimod.F90

```diff
@@ -1087,9 +1087,16 @@ module gsimod
 !      time_window_rad  - upper limit on time window for certain radiance input data
 !      ext_sonde        - logical for extended forward model on sonde data
 !      l_foreaft_thin -   separate TDR fore/aft scan for thinning
+!      hofx_2m_sfcfile  - Calculate h(x) for q2m and T2m from 
```

Comment:
Only select variables in OBS_INPUT are echoed to stdout.  `hofx_2m_sfcfile` and `ignore_2mQM` are echoed to stdout.  It's preferable to echo namelist variables to stdout.  This tells us exactly how gsi.x was configured for a given run.  

We could either
- move these variable to a namelist which is echoed to stdout (e.g., OBSQC)
- add write statements to `gsimod.F90` to print the values of these variables

Reply:
I added write statements to gsimod.

Reply:
Thank you @ClaraDraper-NOAA